clients/csharp-dotnet-core/voice-assistant-test/tool/TurnResult.cs

```diff
@@ -42,11 +42,17 @@ public TurnResult(Turn turn)
         public string ActualRecognizedText { get; set; }
 
         /// <summary>
-        /// Gets or sets the Actual TTS Audio Reponse Duratio (in milliseconds).
+        /// Gets or sets the Actual TTS Audio Response Duration (in milliseconds).
         /// </summary>
         [JsonProperty(Order = 7, DefaultValueHandling = DefaultValueHandling.Ignore)]
         public List<int> ActualTTSAudioResponseDurations { get; set; }
 
+        /// <summary>
+        /// Gets or sets the actual user perceived latency (in milliseconds).
+        /// </summary>
+        [JsonProperty(Order = 8, DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public int UserPerceivedLatency { get; set; }
```

Comment:
Why do you still have both ActualResponseLatency  and UserPerceivedLatency?
There should be only one "expected" and one "actual" latency.
Call them ExpectedUserPerceivedLatency and ActualUserPerceivedLatency

Reply:
Done